lib/elixir_bench_web/schema.ex

```diff
@@ -45,7 +45,7 @@ defmodule ElixirBenchWeb.Schema do
       end
     end
 
-    field :job, non_null(:job) do
+    field :job, (:job) do
```

Comment:
not sure why we removed the `non_null` requirements here? Also, I know it's also done in this file elsewere but instead of `field :name, (:name)` I think `field :name, :name` is preferable :)

Reply:
After another look I saw that this method is related to a benchmark, not the repository. So I think you are right, a benchmark exists as long as a job is registered for it .

Reply:
@PragTob The response body is kinda the same for both, but is more clear when we allow empty objects to be returned because Absinthe just answer with not_found: Here is the response using ```field :job, :job do``` and I think its better.

```
{
   "data" : {
      "job" : null
   },
   "errors" : [
      {
         "locations" : [
            {
               "line" : 1,
               "column" : 0
            }
         ],
         "path" : [
            "job"
         ],
         "message" : "not_found"
      }
   ]
}

```

And here is the response using  ```field :job, non_null(:job do)```
```
{
   "errors" : [
      {
         "path" : [
            "job"
         ],
         "locations" : [
            {
               "line" : 1,
               "column" : 0
            }
         ],
         "message" : "Cannot return null for non-nullable field"
      },
      {
         "message" : "not_found",
         "locations" : [
            {
               "line" : 1,
               "column" : 0
            }
         ],
         "path" : [
            "job"
         ]
      }
   ],
   "data" : null
}
```